src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -127,15 +145,19 @@ export const getJsonFormattedCalendarSync = async (
   return {
     parsedCalendar,
     error: false,
-    errorMessage: null
+    errorMessage: null,
+    timeFrame: Array.isArray(gitHubUsername) ? gitHubUsername[1] : null
```

Comment:
I'm still trying to catch up with the changes here so don't hesitate to correct me if I'm wrong.

I strongly believe `getJsonFormattedCalendarSync()` shouldn't support timeframes for now since it's only used to fetch the default user's calendar. It basically renders the core building block of the calendar(i.e. the "frame") without caring about the timeframes(not to mention it's called "default user" because it has no contributions).

Reply:
On second thought, the same goes for `getJsonFormattedCalendarAsync()`.